.gitignore

```diff
@@ -8,4 +8,6 @@
 *.iml
 .gitreview
 velocity.log
-antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
\ No newline at end of file
+antenna-testing/antenna-sw360-integration-testing/src/test/resources/postgres/sw360pgdb.sql
+.gradle
+**/build
```

Comment:
sorry, same issue here, leading `**/` is always unnecessary.

Reply:
changed it once more :)